plugin/remotecall/main.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tg123/sshpiper/libplugin"
+	"github.com/tg123/sshpiper/plugin/internal/remotecall"
+	"github.com/urfave/cli/v2"
+)
+
+func createRemoteCaller(c *cli.Context) (*remotecall.RemoteCall, error) {
+	remoteCall, err := remotecall.InitRemoteCall(
+		c.String(userClusterMappingEndpoint),
+		c.String(userClusterMappingEndpointToken),
+		c.Generic(remoteAuthEndpoints).(*remotecall.StringMapFlag).Value,
+		c.Generic(remoteAuthEndpointsSecret).(*remotecall.StringMapFlag).Value,
+		c.Generic(remoteEndpoints).(*remotecall.StringMapFlag).Value,
```

Comment:
I think this can panic if type asertion is not successful

Reply:
yeah - its a startup issue of app and thats what i want